config/checkstyle/space-clean

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+git status | grep java | awk '{print $3}' | xargs -o -n 1 vim -c ":retab" -c ":%s/\s\+$//g" -c ":wq" > /dev/null
```

Comment:
Oh, I guess this is the solution to my comment earlier about autoformatting.

Tangentially related: did we decide that are we only developing on linux (since this line uses unix tools)? I occasionally use a windows machine.